### ISSUE TITLE
Challenge 4

### ISSUE THREAD

**Issue by yane3628** [NuevoFoundation/web-basics-curriculum#12]
- On the contact form, we are missing the pic with Benji. This is so it looks like we are building since the beginning.
- Point 4: Add any styling using the example as guideline
		

    - Update: 'Add any CSS styling to the form. This can be font and/or color'
- When sending emails, user does receive an email but the content is not received. Meaning, I get an email

**Comment by Sunny-Dee** [comment]
I added the pic. 
Updated the wording.
For the email setup, you first accept it (to verify in fact you are the owner of the email) and then pretty soon afterwards, they send you the actual form you sent to yourself.
So first, 
![image](https://user-images.githubusercontent.com/6943644/54570072-6c2a1800-499a-11e9-9c1d-52573d6c2cd4.png)

And then you see this:
![exampleform](https://user-images.githubusercontent.com/6943644/54570052-54529400-499a-11e9-9fd5-8293cb8dd689.PNG)
